src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/fly/ModuleFly.kt

```diff
@@ -78,6 +82,11 @@ object ModuleFly : Module("Fly", Category.MOVEMENT, aliases = arrayOf("Glide", "
             FlyVerusB3896Damage,
             FlyVerusB3869Flat,
             FlyNcpClip,
+            
+            FlyMospixelDamageJump,
+            FlyMospixelDamageTimer,
+            FlyMospixelDamageJump,
+            FlyMospixelZoom,
```

Comment:
Why can't we just have one mode with an enum value of which "sub-mode" we want to use? This would reduce the boilerplate of multiple modules in multiple files.